files/package.json

```diff
@@ -38,25 +38,27 @@
     "@babel/plugin-transform-typescript": "^7.25.2",<% } %>
     "@babel/runtime": "^7.25.6",
     "@ember/test-helpers": "^5.2.1",
-    "@embroider/addon-dev": "^8.0.1",
+    "@embroider/addon-dev": "^8.1.0",
     "@embroider/core": "^4.1.0",
     "@embroider/compat": "^4.1.0",
     "@embroider/macros": "^1.18.0",
     "@embroider/vite": "^1.1.5",
     "@eslint/js": "^9.17.0",
     "@glimmer/component": "^2.0.0",<% if (typescript) { %>
-    "@glint/core": "^1.4.0",
-    "@glint/environment-ember-loose": "^1.4.0",
-    "@glint/environment-ember-template-imports": "^1.4.0",
-    "@glint/template": "^1.4.0",
+    "@glint/core": "unstable",
```

Comment:
unstable glint (1.4-unstable) is required because `latest` glint (1.5) does not support `--project`.

`--project` is required for TS to work in this blueprint.